action.yml

```diff
@@ -25,6 +25,9 @@ inputs:
     description: 'Tag of the built image'
     required: false
     default: ''
+  archs:
+    description: 'architecture to build on'
+    required: true
```

Comment:
Can we make it optional and have x86_64 as default list instead? Also, what is the supposed format of this, is it a list?

Reply:
Yes it will be set as optional